### ISSUE TITLE
Install still not working in certain circumstances

### ISSUE THREAD

**Issue by PhilLast** [the-carlisle-group/Acre-Desktop#169]
You'll find that `Install` (`)load .../acre.dws`) as supplied in the latest release (v5.0.12+174) still doesn't work when run in Dyalog v16.0+ on any platform.

This is because it is now attempting to run the enhanced code (now uncommented) but as that has been fixed in each emulator function in v14.0, Dyalog has inserted a flag to force SYNTAX ERRORs on `⎕NPARTS` and `⎕NGET` in later versions where it _should_ run.

I believe running the function that swaps in the appropriate enhancements in place of the emulators within `Install` should settle this once and for all.

Will need someone to `)LOAD` the `acre.dws` into Dyalog for Linux from the new release to be created tonight.

**Comment by e9gille** [comment]
I can help testing once you are ready.

**Comment by PhilLast** [comment]
Fixed (?), committed, pushed and released.

Installed individually and successfully in Windows Dyalog v14.0, v14.1, v15.0, v16.0 & v17.0 by 
`)LOAD .../acre14.0/acre.dws` as supplied in `acre14.0.zip`

@e9gille Yes please!!! - Linux Dyalog ,16.0+ ?

**Comment by e9gille** [comment]
Tested and working on both Mac and Linux (Dyalog v17.0).

**Comment by PhilLast** [comment]
Thanks very much Gil.

Took me long enough but got there in the end.

**Comment by e9gille** [comment]
Your efforts in getting this working just reinforced the importance of specifying engine requirements when it comes to packages. In theory some features (experimental I-beams) may only exist in some engines. But would we want to have to list every engine explicitly?
```json
{
  "name": "Rumba",
  "version": "4.0.1",
  "engine": {
    "Linux-64": ">=16.0",
    "Mac-64": ">=16.0",
    "Windows": ">=16.0",
    "Windows-64": ">=16.0",
  }
} 
```

**Comment by PhilLast** [comment]
Message above re. commit 5.0.14+179 was my mistake. Should have been referencing #170 regarding `⎕SE.acreAPI.itemname`.

This bug was fixed by commit 5.0.13+177
